chesster.js

```diff
@@ -644,6 +647,78 @@ controller.hears([
     });
 });
 
+controller.hears([
+    'pairing ([a-zA-Z0-9]+) ([1-8])'
```

Comment:
We should, absolutely, predefine some filtering class that contains common utilities for parsing. Probably usernames, channels, etc.

We should add an issue for this refactoring.